admin/client.go

```diff
@@ -691,9 +691,15 @@ func (e *GenericOpenAPIError) SetError(errorString string) {
 	e.error = errorString
 }
 
-// format error message using title and detail when model implements Error
-func formatErrorMessage(status, path, method string, v ApiError) string {
-	return fmt.Sprintf("%v %v: HTTP %v (Error code: %q) Detail: %v Reason: %v. Params: %v",
+// FormatErrorMessageWithDetails: format error message using error struct fields: Should be only used for testing
+func FormatErrorMessageWithDetails(status, path, method string, v ApiError) string {
+	detailString := ""
+	if v.BadRequestDetail != nil {
+		badRequestDetail, _ := json.Marshal(v.GetBadRequestDetail())
+		detailString = string(badRequestDetail)
+	}
+
+	return fmt.Sprintf("%v %v: HTTP %v (Error code: %q) Detail: %v Reason: %v. Params: %v, BadRequestDetail: %v",
 		method, path, status, v.GetErrorCode(),
-		v.GetDetail(), v.GetReason(), v.GetParameters())
+		v.GetDetail(), v.GetReason(), v.GetParameters(), detailString)
```

Comment:
knit: we have "details" name for two things, don't know if we could change one of them

Reply:
Done